src/libdrakvuf/win-processes.c

```diff
@@ -1127,29 +1162,41 @@ bool win_find_mmvad(drakvuf_t drakvuf, addr_t eprocess, addr_t vaddr, mmvad_info
         }
         else if (starting_vpn * VMI_PS_4KB <= vaddr && (ending_vpn + 1) * VMI_PS_4KB > vaddr)
         {
+            addr_t pool_tag_delta = is32bit ? 4 : 0xC;
```

Comment:
Please put `4` and `0xC` as `#define`s up top

Reply:
Done